packages/babel-plugin-strip-runtime/src/__tests__/strip-runtime-source-code.test.ts

```diff
@@ -21,21 +23,32 @@ const transformSync = (
     compiledRequireExclude?: boolean;
     run: 'both' | 'bake' | 'extract';
     runtime: 'automatic' | 'classic';
+    extractStylesToDirectory?: { source: string; dest: string };
```

Comment:
Does Babel enforce that `source` and `dest` are defined, or is it just (presumably) typescript?